rqd/rqd/rqconstants.py

```diff
@@ -50,7 +50,8 @@
 
 # RQD behavior:
 RSS_UPDATE_INTERVAL = 10
-RQD_PING_INTERVAL = 60
+RQD_MIN_PING_INTERVAL = 5
+RQD_MAX_PING_INTERVAL = 30
```

Comment:
Could you name it something that makes the unit clear? `RQD_MIN_PING_INTERVAL_SEC` or something like that.

Reply:
Done.